profiler/src/ProfilerEngine/Datadog.Profiler.Native/DllMain.cpp

```diff
@@ -140,11 +140,6 @@ extern "C" HRESULT STDMETHODCALLTYPE DllGetClassObject(REFCLSID rclsid, REFIID r
         return CORPROF_E_PROFILER_CANCEL_ACTIVATION;
     }
 
-#ifdef ARM64
```

Comment:
let's add a flag to still deactivate, maybe something like experimental arm support